docs/.npmrc

```diff
@@ -4,3 +4,13 @@ strict-peer-dependencies=true
 
 # Disable pnpm update notifications since we use corepack to install package managers
 update-notifier=false
+
+@wcp:registry=https://1essharedassets.pkgs.visualstudio.com/1esPkgs/_packaging/WebCompliance/npm/registry/
```

Comment:
What happens if a user tries to install without the necessary environment variables set?

Reply:
Does `process.env.<value>` even work here? I thought this was more of a static file, not something that gets executed in a node context. If that's true, what we need is probably docs saying "in order to work with `@wcp/wcp-consent` you need to add <blablabla> to your `.npmrc` file.

Reply:
@Josmithr They would get a 401 which would fail gracefully, since the package is in optionalDependencies. 

@alexvy86 You're right .npmrc is static. I think you can do `export VAR=foo` and then interpolate it using ${} but adding to .npmrc is the right convention

Reply:
We go to a small amount of trouble to avoid directly including internal Azure artifacts feeds elsewhere in our source code. Could you either use that interpolation or an equivalent `NPM_CONFIG` environment variable in the pipeline that needs it (along with some instructions how to set up the same auth locally) to avoid this here?